docs/solidity-by-example.rst

```diff
@@ -624,9 +610,9 @@ Safe Remote Purchase
             // can call in again here.
             state = State.Inactive;
             // This actually allows both the buyer and the seller to
-            // block the refund.
-            if (!buyer.send(value) || !seller.send(this.balance))
-                throw;
+            // block the refund - the withdraw pattern should be used.
```

Comment:
Maybe make the withdraw patter comment more dominant, with putting it on a new line: `Note: the with...`